orchagent/mirrororch.cpp

```diff
@@ -324,6 +329,48 @@ void MirrorOrch::createEntry(const string& key, const vector<FieldValueTuple>& d
                 m_policerOrch->increaseRefCount(fvValue(i));
                 entry.policer = fvValue(i);
             }
+            else if (fvField(i) == MIRROR_SESSION_SRC_PORT)
+            {
+                auto ports = tokenize(fvValue(i), ',');
+                if (ports.size() != 0)
+                {
+                    for (auto alias : ports)
+                    {
+                        Port port;
+                        if (!gPortsOrch->getPort(alias, port))
+                        {
+                            SWSS_LOG_ERROR("Failed to locate port/LAG %s", alias.c_str());
+                            return;
+                        }
+                    }
+                }
+                entry.src_port = fvValue(i);
+            }
+            else if (fvField(i) == MIRROR_SESSION_DST_PORT)
```

Comment:
For both this and SRC_PORT can we verify that the provided ports are either a physical port or LAG (e.g. not a VLAN or VRF)?

Reply:
Done. Added validation.